jenkins/build_java.gy

```diff
@@ -100,8 +100,6 @@ export PATH="${JAVA_HOME}/bin:$PATH"
 
 cd "${BUILD_DIR}"
 
-touch "local.properties"
```

Comment:
I think that the build will fail unless the local.properties file exists.  It can be empty, but I think it has to be there.